cli/spinner.ts

```diff
@@ -83,7 +83,13 @@ export class Spinner {
     this.#spinner = options?.spinner ?? DEFAULT_SPINNER;
     this.#message = options?.message ?? "";
     this.#interval = options?.interval ?? DEFAULT_INTERVAL;
-    this.#color = options?.color ? COLORS[options.color] : undefined;
+    this.color = options?.color;
+  }
+  set color(value: Color | undefined) {
+    this.#color = value ? COLORS[value] : undefined;
+  }
+  get color() {
+    return this.#color;
```

Comment:
```suggestion
  }
  
  set color(value?: Color) {
    this.#color = value ? COLORS[value] : undefined;
  }
  
  get color() {
    return this.#color;
```
Nits

Reply:
A 'set' accessor cannot have an optional parameter.

Reply:
Oh. Makes sense in hindsight...